news/_posts/releases/2021-08-06-prestashop-1-7-7-6-maintenance-release.md

```diff
@@ -0,0 +1,77 @@
+
+**Back-office**
+- [Preview order - Invoice details is missing the address mail of the customer](https://github.com/PrestaShop/PrestaShop/issues/24482)
+- [It is not possible to create a BYN currency](https://github.com/PrestaShop/PrestaShop/issues/24692)
+- [Product deleteFeatures doesn't delete custom features](https://github.com/PrestaShop/PrestaShop/issues/25163)
+- [Order messages not in customers languages](https://github.com/PrestaShop/PrestaShop/issues/24499)
+- [SQL Manager Form throws FatalThrowableError - Type Error on JOIN SQL query](https://github.com/PrestaShop/PrestaShop/issues/25011)
+- [Order page - The total price excluding taxes is displayed instead of the total price including taxes](https://github.com/PrestaShop/PrestaShop/issues/23957)
+- [View Order - An exception is displayed when changing the order status to payment accepted](https://github.com/PrestaShop/PrestaShop/issues/25195)
+- [Migrated Order page does not contain order reference](https://github.com/PrestaShop/PrestaShop/issues/25078)
+- [Adding payment with comma does not work](https://github.com/PrestaShop/PrestaShop/issues/24103)
+- [Orders page - an exception is displayed when deleting a customer or a guest](https://github.com/PrestaShop/PrestaShop/issues/25319) - CRITICAL
+- [Order message not in order language](https://github.com/PrestaShop/PrestaShop/issues/25362)
+- [View an order - Search endpoint returns HTTP 500](https://github.com/PrestaShop/PrestaShop/issues/25360)
+- [Orders - Problems on product price changing in database](https://github.com/PrestaShop/PrestaShop/issues/24529)
+- [Issue on total when change courier on admin](https://github.com/PrestaShop/PrestaShop/issues/25418)
+- [Orders - An exception is displayed when we add a discount with space in the name](https://github.com/PrestaShop/PrestaShop/issues/24537)
+- [Orders - A modal should'nt be displayed when we add the same product to a specific invoice with updating price](https://github.com/PrestaShop/PrestaShop/issues/24691)
+- [Orders page - Cannot add free shipping discount when Total products = 0](https://github.com/PrestaShop/PrestaShop/issues/24753)
+- [Add new Order page - An exception is displayed when we add a pack](https://github.com/PrestaShop/PrestaShop/issues/24553)
+- [Add new Order - An exception is displayed when ordering a product with text customization - the label contains special expression](https://github.com/PrestaShop/PrestaShop/issues/25194)
+
+
+**Install**
+- [Cannot create an alternative currency after an upgrade](https://github.com/PrestaShop/PrestaShop/issues/24871)
+
+**Front-office**
+- [Brands are shown wrongly in the FO](https://github.com/PrestaShop/PrestaShop/issues/25214)
+- ['Total available for each user' in cart rule not considered if the customer is signed in from the checkout page](https://github.com/PrestaShop/PrestaShop/issues/25203)
+- [Smarty template isCached function bug](https://github.com/PrestaShop/PrestaShop/issues/24235) - CRITICAL
```

Comment:
Suggestion: move critical bugfixes into a dedicated section, and provide context about why they are critical (maybe remind our labels definitions)

Reply:
Ok, done 👍